plugin/src/main/java/com/denizenscript/denizen/objects/PolygonTag.java

```diff
@@ -747,6 +780,12 @@ public static void register() {
         // @returns PolygonTag
         // @description
         // Returns a copy of the polygon, with the specified Y value included as part of the Y range (expanding the Y-min or Y-max as needed).
+        // @example
+        // # Notes "my_polygon" as "my_new_polygon" but with the player's Y location included in the Y range.
+        // # For example, if "my_polygon" has a maximum-Y of 80 and a minimum-Y of 60 and the player's Y location was 95,
+        // # then "my_new_polygon" will have a maximum-Y of 95 and a minimum-Y of 60. However, if the player's Y location
+        // # was 50, then the maximum-Y value would stay the same because that Y location is already included within the Y range.

```

Comment:
and the minimum Y will change instead

Also please don't do block newlines like this, cut newlines between sentences not at random